config/archiver.conf

```diff
@@ -0,0 +1,52 @@
+##### Archiver properties #####
```

Comment:
it would be good to keep the same standard adopted by the other confs, that is, a comment and an example (or default value) on top of each property, check [here](https://github.com/ufcg-lsd/saps-engine/blob/develop/config/dispatcher.conf)